axisApp/Db/axis_base.template

```diff
@@ -123,7 +123,9 @@ record(stringin, "$(P)$(AXIS):MOTOR") {
 
 record(bo, "$(P)$(AXIS):STOP") {
   field(DESC, "Stop the axis motor")
-  field(OUT, "$(P)$(mAXIS).STOP 1")
+  field(OUT, "$(P)$(mAXIS).STOP")
```

Comment:
Do you need PP ?

Reply:
It doesn't seem to be a problem when testing, and it's talking to a separate IOC, I though PP was ignored when talking to another IOC?

Reply:
It turns out using PP here does fix a strange issue where if you stop the axis and then write a new setpoint the setpoint is ignored (but if you then set the setpoint again the motor moves).

Reply:
With a separate IOC PP is ignored and the effective "pp" of the destination field is used, but there is no harm including it and it protects you if the db was used for a motor in the same IOC.